src/Chunk.h

```diff
@@ -392,18 +392,37 @@ class cChunk :
 	{
 		return m_ChunkData.GetMeta(a_RelX, a_RelY, a_RelZ);
 	}
-	inline void       SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
+	inline void  SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
```

Comment:
>With the added SetMetaQuiet method, would it be worth having the SetMeta method just call SetMetaQuiet with the boolean arguments set to true?

- Or, Just add the two extra parameters to the regular `SetMeta` and make them default to true, this eliminates the need for metaQuiet -